### ISSUE TITLE
question about train/test spllit

### ISSUE THREAD

**Issue by HiromasaYamanishi** [yoonjong12/pinsage#8]
Thank you for great work!
I have a question about train/test split. In original dgl process_movielens.py, whole graph g is split by train_test_split_by_time and train_g is created as a subpart of whole graph g, and used as train data. Although, In process_wine.py, Whole graph g is used as train_graph. Is there no warry about accessing test node in train time?

**Comment by HiromasaYamanishi** [comment]
After splitted g into train_g, the accuracy is still same high. I'm maybe misunderstanding something..

**Comment by yoonjong12** [comment]
Thank you for your feedback. I checked and found that actually my code doesn't use train_test_split_by_time properly. I'm going to repair the code as a whole soon, and I think it'll take a while.
